files/organizations/allowed_regions.json.tpl

```diff
@@ -54,13 +75,17 @@
             "Resource": "*",
             "Condition": {
                 "StringNotEquals": {
-                    "aws:RequestedRegion": ${jsonencode(allowed)}
+                    "aws:RequestedRegion": ${jsonencode(allowed)
+                    }
+                }
+                %{ if length(exceptions) > 0 ~
+                },
+                "ArnNotLike": {
+                    "aws:PrincipalARN": ${jsonencode(exceptions)
+                    }
```

Comment:
```suggestion
                    "aws:RequestedRegion": ${jsonencode(allowed)}
                }
                %{ if length(exceptions) > 0 ~ },
                "ArnNotLike": {
                    "aws:PrincipalARN": ${jsonencode(exceptions)}
```
Personally I think this is a better readable syntax

Reply:
Or replace the whole block with this?
```
            "Condition": {
                "StringNotEquals": {
                    "aws:RequestedRegion": ${jsonencode(allowed)}
                %{ if length(exceptions) > 0 ~ }
                },
                "ArnNotLike": {
                    "aws:PrincipalARN": ${jsonencode(exceptions)}
                %{ endif ~ }
                }
            }
```

Reply:
autoformatter, i've reverted the autoformat